### PR TITLE
Improve fybrik website

### PR DESCRIPTION
**Pull request by simanadler**
Addresses fybrik website improvements #1040

Signed-off-by: simanadler <sima@il.ibm.com>

### CODE REVIEW

File: site/docs/concepts/connectors.md [hunk 1]
```diff
@@ -24,18 +24,18 @@ In addition, to benefit from the [control plane security](../tasks/control-plane
...
 
 Fybrik is not a data catalog. Instead, it links to existing data catalogs using connectors.
 The default installation of Fybrik installs [Katalog](../reference/katalog.md), a built-in data catalog using Kubernetes CRDs used for evaluation. A connector to [ODPi Egeria](https://www.odpi.org/projects/egeria) is also available.
 
 ### Policy manager
 
 Enforcing data governance policies requires a Policy Decision Point (PDP) that dictates what enforcement actions need to take place.
-Fybrik supports a wide and extendible set of enforcement actions to perform on data read, write or copy. These include transformation of data, verification of the data, and various restrictions on the external activity of an application that can acceess the data.
+Fybrik supports a wide and extendible set of enforcement actions to perform on data read, copy, (future) write or delete. These include transformation of data, verification of the data, and various restrictions on the external activity of an application that can acceess the data.
```
Comment (shlomitk1) [inline site/docs/concepts/connectors.md:35]: typo: access

File: site/docs/concepts/connectors.md [hunk 2]
```diff
@@ -24,18 +24,18 @@ In addition, to benefit from the [control plane security](../tasks/control-plane
...
 
 A PDP returns a list of enforcement actions given a set of policies and specific context about the application and the data it uses. 
 Fybrik includes a PDP that is powered by [Open Policy Agent](https://www.openpolicyagent.org/) (OPA). However, the PDP can also use external policy managers via connectors, to cover some or even all policy types. 
 
-Policies are therefore defined externally in the policy manager of choice. Fybrik provides a package to help writing data policies in OPA. Otherwise, data stewards are expected to keep using the policy manager that they already use, as long as there is a connector to it.
+Policies are therefore defined externally in the data governance policy manager of choice. Fybrik provides a package to help writing data policies in OPA. Otherwise, data stewards may continue using the data governance policy manager that they already use, assuming there is a Fybrik connector for it.
```
Comment (shlomitk1) [inline site/docs/concepts/connectors.md:40]: This is not clear ('already' is misleading). Sounds as if Fybrik provides a package to write policies in OPA on top of what is used already. I think we no longer support several policy managers.

File: site/docs/concepts/introduction.md [hunk 3]
```diff
@@ -4,10 +4,9 @@ Fybrik is a cloud native platform to unify data access and governance, enabling
 
 Fybrik allows:
 
-* **Data users** to use data in a self-service model without manual processes and without dealing with credentials. Use common tools and frameworks for reading from and exporting data to data lakes or data warehouses.
-* **Data stewards** to control data access and data usage by applications. Use the organization's _policy manager_ and _data catalog_ of choice and let Fybrik enforce data usage policies even after data is accessed.
-* **Data operators** to automate data lifecycle mangement via implicit data copies, eliminating the need for manual versioning and copying of data.
-
+* **Data users** to use data in a self-service model without manual processes, without neededing to confer with data stewards, and without dealing with credentials. Use common tools and frameworks for reading from and exporting data to data lakes or data warehouses.
```
Comment (shlomitk1) [inline site/docs/concepts/introduction.md:7]: typo: needing

File: site/docs/concepts/introduction.md [hunk 4]
```diff
@@ -19,31 +18,35 @@ The inputs to Fybrik are declarative definitions with separation of aspects:
...
-- Makes data available in locations where it is needed. Thus in a multi cluster scenario it may copy data from one location to another
+- Enforces governance relating to the data and its lifecycle; including limiting what data the business logic can access, performing transformations as needed, controlling what the business logic can export and to where
+- Makes data available in locations where it is needed. Thus in a multi cluster scenario it may copy data from one location to another, something known as an implicit copy.  The implicit copy is deleted when no longer needed.
 
 Fybrik is an open solution that can be extended to work with a wide range of tools and data stores. For example, the injectable [modules](./modules.md) and the [connectors](./connectors.md) to external systems (e.g., to a data catalog) can all be third party.
 
+The logic used by fybrik to generate the data planes is customizable.  An organization can determine how best its infrastructure should be leverage via [config policies](./config-policies.md)
```
Comment (shlomitk1) [inline site/docs/concepts/introduction.md:30]: be leveraged?

File: site/docs/concepts/introduction.md [hunk 5]
```diff
@@ -19,31 +18,35 @@ The inputs to Fybrik are declarative definitions with separation of aspects:
...
+
 ## Applications
 
-Fybrik considers applications as first level entities. Before running a workload, an application needs to be registred to a Fybrik control plane by applying a `FybrikApplication` resource. This is the declarative definition provided by the data user. The registration provides context about the application such as the purpose for which it's running, the data assets that it needs, and a selector to identify the workload. Additional context such as geo-location is extracted from the platform. 
+Fybrik considers applications as first level entities. Before running a workload, an application needs to be registred to a Fybrik control plane by applying a `FybrikApplication` resource. This is the declarative definition provided by the data user. The registration provides context about the application such as the purpose for which it's running, the data assets that it accesses, and a selector to identify the workload. Additional context such as geo-location is extracted from the platform.
```
Comment (shlomitk1) [inline site/docs/concepts/introduction.md:34]: typo: registered

File: site/docs/concepts/introduction.md [hunk 6]
```diff
@@ -19,31 +18,35 @@ The inputs to Fybrik are declarative definitions with separation of aspects:
...
 
+There are specific situations in which there is no workload associated with a `FybrikApplication` resource.  Examples of these are requests to ingest data into a governed environment, or (future) requests to clean up data in the governed environment based on data governance policies.
+
 ## Security
 
-While the Fybrik handles enforcement of data governance policies, if one could access the data not through the platform then we lose control over data usage.
+While Fybrik handles enforcement of data governance policies, if one could access the data not through the platform then we lose control over data usage.
 
-For this reason, Fybrik does not let user applications ever observe data access credentials, both for externally created data assets and for data assets created by the Fybrik control plane and applications running in it.
+For this reason, Fybrik does not let user applications ever observe data access credentials, both for externally created data assets nor for data assets created by the Fybrik control plane and applications running in it.
```
Comment (shlomitk1) [inline site/docs/concepts/introduction.md:44]: both -> neither?

File: site/docs/concepts/introduction.md [hunk 7]
```diff
@@ -19,31 +18,35 @@ The inputs to Fybrik are declarative definitions with separation of aspects:
...
 
 ## Multicluster
 Fybrik supports data paths that access data stores that are external to the cluster such as cloud managed object stores or databases as well as data stores within the cluster such as databases running in Kubernetes. All applications and modules however will run within a cluster that has Fybrik installed.
 
-Multi-cloud and hybrid cloud scenarios are supported out of the box by running Fybrik in multiple Kubernetes clusters and configuring the manager to use a multi cluster coordination mechanism such as razee. This enables cases such as running transformations on-prem while creating an implicit copy of an on-prem SoR table to a public cloud storage system.
+Multi-cloud and hybrid cloud scenarios are supported out of the box by running Fybrik in multiple Kubernetes clusters and configuring the manager to use a multi cluster coordination mechanism such as razee. This enables cases such as running, for example, transformations on-prem while creating an implicit copy of an on-prem SoR table to a public cloud storage system.
```
Comment (shlomitk1) [inline site/docs/concepts/introduction.md:51]: multi cluster -> multi-cluster ?

File: site/docs/concepts/introduction.md (hunk 3 above)
Comment (simanadler) [reply site/docs/concepts/introduction.md]: fixed

File: site/docs/concepts/connectors.md (hunk 2 above)
Comment (simanadler) [reply site/docs/concepts/connectors.md]: fixed

File: site/docs/concepts/introduction.md (hunk 7 above)
Comment (simanadler) [reply site/docs/concepts/introduction.md]: fixed

File: site/docs/concepts/introduction.md (hunk 6 above)
Comment (simanadler) [reply site/docs/concepts/introduction.md]: fixed

File: site/docs/concepts/introduction.md (hunk 5 above)
Comment (simanadler) [reply site/docs/concepts/introduction.md]: fixed

File: site/docs/concepts/introduction.md (hunk 4 above)
Comment (simanadler) [reply site/docs/concepts/introduction.md]: fixed

**Review by shlomitk1 — APPROVED** [review]

**Review by roytman — APPROVED** [review]

**Pull request merged**